loopin_test.go

```diff
@@ -220,11 +265,12 @@ func testLoopInTimeout(t *testing.T,
 	} else {
 		// Create an external htlc publish tx.
 		var pkScript []byte
-		if outputType == swap.HtlcNP2WSH {
-			pkScript = s.htlcNP2WSH.PkScript
+		if !inSwap.isTaproot() {
```

Comment:
No, let me clarify. When taproot and musig is active we'll use p2tr for both external and internal case. Otherwise p2wsh is internal and np2wsh is external so no changes.

Reply:
ah ok, fixed